source/wp-content/themes/wporg-documentation-2022/theme.json

```diff
@@ -1,7 +1,16 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/theme.json",
 	"version": 2,
-	"settings": {},
+	"settings": {
+		"custom": {
+			"alignment": {
+				"aligned-max-width": "100%"
```

Comment:
Do we use this value anywhere? IIRC it's leftover from the long-past (news theme), but i don't recall if we started using it again in devhub

Reply:
Not that I know of. We keep needing to override it. I'll test removing it in my sandbox.

Reply:
It's used [here](https://github.com/WordPress/wporg-parent-2021/blob/trunk/source/wp-content/themes/wporg-parent-2021/sass/base/_layout.scss#L26), but removing this doesn't seem to affect any of the sites I've tested. It is being overridden in all the sites I've worked on (Dev Res, Docs and Learn), so I think it can be safely removed. I think not in this PR though.

Reply:
Created an [issue](https://github.com/WordPress/wporg-parent-2021/issues/141)

Reply:
> I think not in this PR though.

Why add it now just to remove it later?

Reply:
Fair point 😁 It's just that I've already got 4 PRs to merge for this one fix, and I feel like it needs more testing.

Reply:
I've put in in progress on the [Learn board](https://github.com/orgs/WordPress/projects/40/views/26), I will continue with it